sources/packages/backend/libs/services/src/constants/system-configurations-constants.ts

```diff
@@ -90,3 +90,8 @@ export const BLOCKED_DISBURSEMENT_NOTIFICATION_MIN_DAYS_INTERVAL = 7;
  * Represents the maximum number of disbursement blocked notifications to send.
  * */
 export const BLOCKED_DISBURSEMENT_MAXIMUM_NOTIFICATIONS_TO_SEND = 3;
+
+/**
+ * Bull Dashboard route.
+ */
+export const BULL_BOARD_ROUTE = "admin/queues";
```

Comment:
the route needs in `/` in the end on open shift environments. 

![image](https://github.com/user-attachments/assets/2271f9ac-4258-4e92-8d19-82b5c4cd9132)

Reply:
This configuration is not new and was just moved. I believe that if it was wrong it was already causing some issues. Does it make sense?